packages/roosterjs-editor-dom/test/DomTestHelper.ts

```diff
@@ -113,3 +113,12 @@ export function itFirefoxOnly(
     const func = __karma__.config.browser == 'Chrome' ? xit : it;
     return func(expectation, assertion, timeout);
 }
+
+export function itChromeOnly(
```

Comment:
nit: possible typo? isChromeOnly?

Reply:
no, "it" is the function name of running a test case.